product-development/README.md

```diff
@@ -0,0 +1,83 @@
+
+### Free-form discussion in Clubhouse stories
+All Tilters are encouraged to collaborate in _discussing_ general ideas in a free-form fashion, with no strict process, no structure, no specific timelines, and no specific implementation commitments. Tilters should prefer discussing ideas in Clubhouse stories, linking ideas together, thus building a web of good ideas that are easily searchable for posterity.
+
+### Defining high-level business initiatives with Clubhouse epics
+All Tilters are encouraged to collaborate in _definining_ high-level business initiatives. Similar to general ideas, there is no strict process for collaboration, except that each business initiative should be documented as a Clubhouse epic. As an epic becomes increasingly well-defined over time, it should have more and more of these components with increasing fidelity:
+- Problem statement
+- Solution scope
+- Attached stories with relevant ideas, including designs
+
+Victor is responsible for ensuring that the backlog of epics is continually full and that epics are increasingly being better defined. There should be no shortage of epics.
+
+## Picking an epic to work on
+When a Tilt engineer is free, they should pick a high priority epic to work on and make themself the _sole_ owner of the epic. The engineer should find other folks to collaborate on the epic as needed, but they are the [DRI](https://medium.com/@mmamet/directly-responsible-individuals-f5009f465da4) (directly responsible individual) for driving the epic to completion. There should be at most one owner on any epic, and the owner is the DRI.
+
+The epic may have a fairly broad initial business problem. The engineer(s) should carve out and define a smaller problem, and spec out a solution that can be accomplished within a target of **_7 working days_** (counted from when the epic moves into `In Progress`, more below on states). If the solution cannot be finished within 7 days, keep reducing scope. The epic description should be updated with the smaller scope and acceptance criteria, and the engineer(s) should create additional epics and/or stories to capture the future work.
+
+The purpose of a single epic is _not_ to completely solve a business problem. Rather, completing an epic should likely achieve incremental progress at addressing a single problem, adding incremental user value as a result. In some cases, any realized user value may only appear in future epics. 
+
+After the epic is completed, the engineer should find the next epic to work on, either as a DRI owner, or as a collaborator. Engineers should self-organize and coordinate timelines ad hoc when picking epics. There are no stable teams, no timeboxed sprints, and no synced sprints across teams. Engineers should avoid working on multiple epics concurrently. 
```

Comment:
re: collaborators: do you expect teams of collaborators to change with every new epic? Or do you expect that people will organize into teams of 2-3 and decide on the next epic together?

Reply:
I don't have a specific desired outcome or guess as to how people will form teams if they are given this freedom. I do want to specifically afford the freedom to people because it should allow Tilt to be more responsive to business needs, without having to "break the process", which often leads to people getting frustrated.

I updated this text here to say a person should be a DRI for at most _one_ epic at a given time. But that they could actually be working on multiple epics at the same time. This way, there's always one person who should be driving forward progress for the given epic, and can help "project manage" for lack of better term. And I think that a person shouldn't be doing these DRI responsibilities for multiple epics at the same time, since context switching for project management and head space is very difficult imo. But I did want to allow folks to work on multiple epics at the same time, if they simply have the capacity to do so.